tests/test_pipestat.py

```diff
@@ -2513,3 +2516,188 @@ def test_select_history_complex_objects(
             assert len(history_result.keys()) == 1
             assert "output_image" in history_result
             assert len(history_result["output_image"].keys()) == 2
+
+
+@pytest.mark.skip(reason="requires pephub login to function")
+class TestPEPHUBBackend:
+    """
+    THESE TESTS WILL FAIL IF YOU ARE NOT SIGNED IN TO PEPHUB
+
+    use `phc login` to sign in.
+    """
+
```

Comment:
Overall my oppinion, we shouldn't test it on deployed version of pephub

Reply:
As discussed, we will leave these tests but have them continue skipping. In the future, we should consider doing testing on a development instance of PEPhub instead of production.